chia/_tests/util/misc.py

```diff
@@ -638,49 +630,3 @@ def __ge__(self: T_ComparableEnum, other: T_ComparableEnum) -> object:
             return NotImplemented
 
         return self.value.__ge__(other.value)
-
-
-async def add_blocks_in_batches(
-    blocks: list[FullBlock],
-    full_node: FullNode,
-    header_hash: Optional[bytes32] = None,
-) -> None:
-    if header_hash is None:
-        diff = full_node.constants.DIFFICULTY_STARTING
-        ssi = full_node.constants.SUB_SLOT_ITERS_STARTING
-        fork_height = -1
-        fork_info = ForkInfo(-1, fork_height, full_node.constants.GENESIS_CHALLENGE)
-    else:
-        block_record = await full_node.blockchain.get_block_record_from_db(header_hash)
-        assert block_record is not None
-        ssi, diff = get_next_sub_slot_iters_and_difficulty(
-            full_node.constants, True, block_record, full_node.blockchain
-        )
-        fork_height = block_record.height
-        fork_info = ForkInfo(block_record.height, fork_height, block_record.header_hash)
-
-    _, dummy_node_id = await add_dummy_connection(full_node.server, "127.0.0.1", 12315)
-    dummy_peer = full_node.server.all_connections[dummy_node_id]
```

Comment:
note that I removed the `dummy_peer`, as that would have pulled in even more test code

Reply:
changes to test code -> tests passing == checks out to me